client-app/core/api/graphql/config/cache-persistor.ts

```diff
@@ -0,0 +1,35 @@
+import { CachePersistor, LocalStorageWrapper } from "apollo3-cache-persist";
+import { cache } from "@/core/api/graphql/config/cache";
+import type { ApolloCache } from "@apollo/client/core";
+import type { ApolloPersistOptions } from "apollo3-cache-persist/lib/types";
+
+class ThemeCachePersistor<T> extends CachePersistor<T> {
+  private originalApolloCache: ApolloCache<T>;
+  private originalApolloGc: ApolloCache<T>["gc"];
+
+  constructor(options: ApolloPersistOptions<T>) {
+    super(options);
+
+    // Workaround because of bug: authors of apollo3-cache-persist forgot to call save state on gc method
```

Comment:
pull request to apollo3-cache-persist please

Reply:
https://github.com/apollographql/apollo-cache-persist/pull/515